lightning-network-book.tex

```diff
@@ -240,36 +240,36 @@
 \newglossaryentry{penalty tx}{name={Penalty Transaction},
   description={look at breach remedy transaction.}}
 \newglossaryentry{lnd}{name={lnd},
-  description={Implementation of the Lightning Network Protocol by the San Francisco based company Lightning Labs.
+  description={Implementation of the Lightning Network Protocol by the San Francisco based company Lightning Labs (\url{https://lightning.engineering}). Source code is at \url{https://github.com/lightningnetwork/lnd}.
 It is written in Go.}}
 \newglossaryentry{eclair}{name={Eclair},
-  description={Implementation of the Lightning Network Protocol by the Paris based company ACINQ. It is written in Scala.}}
+  description={Implementation of the Lightning Network Protocol by the Paris based company ACINQ (\url{https://acinq.co}). Source code is at \url{https://github.com/ACINQ/eclair}. It is written in Scala.}}
 \newglossaryentry{clightning}{name={c-lightning},
-  description={Implementation of the Lightning Network Protocol by the Victoria based Blockstream. It is written in C.}}
+  description={Implementation of the Lightning Network Protocol by the Victoria based Blockstream(\url{https://blockstream.com}). Source code is at \url{https://github.com/ElementsProject/lightning}. It is written in C.}}
 \newglossaryentry{shachain}{name={Sha-chain},
   description={}}
 \newglossaryentry{hdwallet}{name={HD Wallet},
   description={}}
 \newglossaryentry{bip}{name={BIP},
-  description={}}
+  description={BIP is short for Bitcoin Improvement Proposals. It is a mechanism by which proposals for the improvement of the Bitcoin network can be made. Submitted proposals are discussed heavily and the best one implemented. See \url{https://github.com/bitcoin/bips}.}}
```

Comment:
best ones are implemented?

Reply:
```suggestion
  description={BIP is short for Bitcoin Improvement Proposals. It is a mechanism by which proposals for the improvement of the Bitcoin network can be made. Submitted proposals are discussed heavily and the best ones are implemented. See \url{https://github.com/bitcoin/bips}. Currently 34 of the 121 BIPs have been finalized.}
```

Reply:
there is a } missing at the end, oops...